internal/core/config.go

```diff
@@ -313,7 +313,7 @@ func ParseConfig(content []byte, path string, parentConfig Config, isGlobal bool
 		config.Note.Extension = note.Extension
 	}
```

Comment:
@WhyNotHugo 
It occurred to me that if there is an error, then we will cease filling the rest of the config entirely.
This didn't occur to me when reviewing #431. I guess it's acceptable, as `user.Current()` should always be resolvable, otherwise something is seriously wrong?

Otherwise, we could just wrap the error and continue? What would be the drawback of that approach?